### PR TITLE
provide userOwnsProject to gui

### PR DESCRIPTION
**Pull request by benjiwheeler**
help gui know what to show by telling it if the user owns the project

### CODE REVIEW

File: src/lib/storage.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ class Storage extends ScratchStorage {
             projectAsset => {
                 const [projectId, revision] = projectAsset.assetId.split('.');
                 return revision ?
-                    `${PROJECT_SERVER}/internalapi/project/${projectId}/get/${revision}` :
-                    `${PROJECT_SERVER}/internalapi/project/${projectId}/get/`;
+                    `${PROJECT_HOST}/internalapi/project/${projectId}/get/${revision}` :
+                    `${PROJECT_HOST}/internalapi/project/${projectId}/get/`;
```
Comment (rschamp) [inline src/lib/storage.js]: This looks unrelated but I don't think it's an issue if it goes in.

**Comment by benjiwheeler** [comment]
I hear you. I think I can implement a system of capabilities like that, but it seems like getting that right is nontrivial, and shouldn't be a blocker for this round of gui changes. I think the best way to move quickly towards the point where the top level API of gui doesn't change is to iterate quickly now.

**Comment by benjiwheeler** [comment]
FYI I created this issue: https://github.com/LLK/scratch-www/issues/2168

**Comment by rschamp** [comment]
(blocked until https://github.com/LLK/scratch-gui/pull/3303 is merged/built)

**Comment by benjiwheeler** [comment]
I am holding off on this, and will approach it in a new way after #3303 is merged.

**Pull request closed without merging**